cmd/tailscale/cli/cli.go

```diff
@@ -104,6 +104,23 @@ func newFlagSet(name string) *flag.FlagSet {
 	return fs
 }
 
+// CleanUpArgs rewrites command line arguments for simplicity and backwards compatibility.
+// In particular, it rewrites --authkey to --auth-key.
+func CleanUpArgs(args []string) []string {
+	out := make([]string, 0, len(args))
+	for _, arg := range args {
+		// Rewrite --authkey to --auth-key, and --authkey=x to --auth-key=x,
+		// and the same for the -authkey variant.
+		if strings.HasPrefix(arg, "--authkey") || strings.HasPrefix(arg, "-authkey") {
```

Comment:
technically this should match `/^-{1,2}authkey\b/` and this is missing the `\b` part; it matches `--authkeyfoo` still.

Probably not a problem in practice. At least yet. :)

Reply:
I think the only relevant `\b` here is `=`. (This is argv, not shell, so spaces don't get any magic treatment.) Fixed.